browser/src/control/Control.Mention.ts

```diff
@@ -59,23 +59,34 @@ interface PopupData {
 	id: string;
 }
 
-L.Control.Mention = L.Control.extend({
-	onAdd: function(map: ReturnType<typeof L.map>) {
+interface MessageEvent {
+    typingMention: boolean;
+}
+
+class Mention {
+	map: ReturnType<typeof L.map>;
+	newPopupData: PopupData;
+	firstChar: string;
+	users: any;
+	itemList: Array<any>;
+	_map: ReturnType<typeof L.map>;
```

Comment:
we don't need duplicated `map` and `_map` -> please leave only one: `map` and use it everywhere